ironic-deployment/keepalived/keepalived_patch.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         name: ironic-endpoint-keepalived
         securityContext:
               capabilities:
-                add: ["NET_ADMIN"]
+                add: ["NET_ADMIN", "NET_RAW"]
```

Comment:
Is there any other alternative to avoid adding this capability in order to fix the issue with ironic-endpoint? The reason I was curious is mainly that this might be a [security flaw](https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/com.vmware.nsxt.ncp_kubernetes.doc/GUID-AC96C51A-052B-403F-9670-67E55C4C9170.html#disallow-netadmin-and-netraw-capabilities-of-untrusted-containers-2).

Reply:
@furkatgofurov7 I am having difficulty understanding how adding these two capabilities is a security flaw, Yes you should not add these capabilities to untrusted containers, but for your own trusted containers, if you dont add them and if your container needs them, it will simply not work.

Reply:
Okay, I was not sure, if this is in the context of `untrusted containers`, my understanding was, it can be any container.